### ISSUE TITLE
batchnormalization for wgan2-keras.ipynb

### ISSUE THREAD

**Issue by jq460494839** [tjwei/GANotebooks#9]
why in Discriminator, you didn't use batchnormalization？
